### DISCUSSION TITLE
[QUESTION] Is it just the documentation that isn't finished?

### DISCUSSION THREAD

**Discussion by secondmanveran** [themesberg/flowbite-vue#197] [Q&A] [answered]
Hello and thanks for this package.

On the readme it says that the documentation isn't finished. Is it just the docs, or are the components not finished as well?

**Comment by cogor** [comment, marked as answer]
Hello, not all components are finished, but we are actively working on them. Incomplete components are marked with a "-" prefix in the documentation sidebar. Consequently, the documentation for these components is also not ready.

**Comment by secondmanveran** [comment]
@cogor thanks a lot for the update. Is there any sort of ETA for the components?

**Comment by cogor** [comment]
I think it(sidebar) will definitely be ready this month.

**Comment by secondmanveran** [comment]
Oh thats great news! Thanks again. Is there a mailing list I can get on so that I can get an update when it's released?

**Comment by cogor** [comment]
I'm sorry, I confused your question with a question from another issue. It's hard to say when all the components will be ready since there is an issue for each component. You can subscribe to the ones you're interested in and track the progress on them.

**Comment by cogor** [comment]
> Oh thats great news! Thanks again. Is there a mailing list I can get on so that I can get an update when it's released?

I'm not sure if we have a mailing list, but I will definitely comment on this issue when everything is ready, and you will receive a notification via email from GitHub. Additionally, we have a Discord where we will also share such news.

**Comment by secondmanveran** [comment]
Cool thanks
